.changeset/poor-yaks-think.md

```diff
@@ -0,0 +1,7 @@
+---
+"@frameless/pdc-dashboard": minor
+"@frameless/vth-dashboard": minor
+"@frameless/strapi-plugin-language": minor
+---
+
+Maak een taal-Strapi-plugin die helpt bij het toevoegen van taalkenmerken aan de Strapi-blok. Dit zal de toegankelijkheid voor ondersteunende technologieën zoals schermlezers verbeteren, waardoor bezoekers van het Digitaal Loket gemakkelijker en effectiever kunnen navigeren en gebruik kunnen maken van de inhoud.
```

Comment:
"Maak een taal-Strapi-plugin die helpt bij het toevoegen van taalkenmerken aan de Strapi-blok." vertelt meer de opdracht die de developer had.

Wat denk je van:
Wanneer een gedeelte van de pagina in een andere taal is geschreven, dan kun je nu in Strapi de taal instellen. Als je dit doet verbeter je de toegankelijkheid, want dan kan de tekst met de juiste taal voorgelezen worden.